permissions/plugin-adobe-cloud-manage

```diff
@@ -0,0 +1,7 @@
+---
+name: "adobe-cloud-manager"
+github: "jenkinsci/adobe-cloud-manager-plugin"
+paths:
+- "org/jenkins-ci/plugins/adobe-cloud-manager"
```

Comment:
```suggestion
- "io/jenkins/plugins/adobe-cloud-manager"
```

See https://github.com/jenkinsci/adobe-cloud-manager-plugin/blob/master/pom.xml#L10

Reply:
Ahh! great catch! thank you!